packages/nest/src/open-feature.module.ts

```diff
@@ -17,7 +26,18 @@ import { EvaluationContextInterceptor } from './evaluation-context-interceptor';
 export class OpenFeatureModule {
   static forRoot({ useGlobalInterceptor = true, ...options }: OpenFeatureModuleOptions): DynamicModule {
     OpenFeature.setTransactionContextPropagator(new AsyncLocalStorageTransactionContext());
-    const providers: NestProvider[] = [];
+
+    if (options.logger) {
```

Comment:
I'm wondering if there's a way to pass NetsJS' global logger and inject it instead of passing it explicitly as an option.

Do you think it is possible using the current structure?

Reply:
Had a look, it seems not to be possible to get it.
Maybe we could just add a facade for Nest.js loggers.